src/commands/secret/mod.rs

```diff
@@ -108,12 +151,29 @@ pub fn delete_secret(name: &str, user: &GlobalUser, target: &Target) -> Result<(
     let response = client.request(&DeleteSecret {
         account_identifier: &target.account_id,
         script_name: &target.name,
-        secret_name: name,
+        secret_name: name.clone(),
     });
 
     match response {
-        Ok(_) => message::success(&format!("You've deleted the secret {}.", name)),
-        Err(e) => failure::bail!(format_error(e)),
+        Ok(_) => message::success(&format!("Success! Deleted secret {}.", name)),
+        Err(e) => match upload_draft_worker(&e, user, target) {
```

Comment:
it doesn't make sense to create a  worker  in this scenario; we should use normal error messages here.